src/agent.py

```
@@ -259,59 +310,145 @@ async def fetch_long_term():
+    
+    Returns list of facts for compatibility with the tracked_db pattern.
+
+    Args:
+        query: The user query
+        embedding_store: The embedding store to search
+
+    Returns:
+        List of retrieved facts
+    """
+    logger.debug(f"Starting long-term fact retrieval for query: {query}")
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)